test/regress/expect/tpch1/q01.txt

```diff
@@ -0,0 +1,37 @@
+select
+	l_returnflag,
+	l_linestatus,
+	sum(l_quantity) as sum_qty,
+	sum(l_extendedprice) as sum_base_price,
+	sum(l_extendedprice * (1 - l_discount)) as sum_disc_price,
+	sum(l_extendedprice * (1 - l_discount) * (1 + l_tax)) as sum_charge,
+	avg(l_quantity) as avg_qty,
+	avg(l_extendedprice) as avg_price,
+	avg(l_discount) as avg_disc,
+	count(*) as count_order
+from
+	lineitem
+where
+	l_shipdate <= date '1998-12-01' - interval '90' day
+group by
+	l_returnflag,
+	l_linestatus
+order by
+	l_returnflag,
+	l_linestatus
+Total cost: 11882428.35
+PhysicOrder  (inccost=11882428.35, cost=11.35, rows=6) (actual rows=4)
+    Output: l_returnflag[0],l_linestatus[1],{sum(l_quantity)}[2],{sum(l_extendedprice)}[3],{sum(l_extendedprice*1-l_discount)}[4],{sum(l_extendedprice*1-l_discount*1+l_tax)}[5],{avg(l_quantity)}[6],{avg(l_extendedprice)}[7],{avg(l_discount)}[8],{count(*)(0)}[9]
+    Order by: l_returnflag[0], l_linestatus[1]
+    -> PhysicHashAgg  (inccost=11882417, cost=5881202, rows=6) (actual rows=4)
+        Output: {l_returnflag}[0],{l_linestatus}[1],{sum(l_quantity)}[2],{sum(l_extendedprice)}[3],{sum(l_extendedprice*1-l_discount)}[4],{sum(l_extendedprice*1-l_discount*1+l_tax)}[5],{avg(l_quantity)}[6],{avg(l_extendedprice)}[7],{avg(l_discount)}[8],{count(*)(0)}[9]
+        Aggregates: sum(l_quantity[2]), sum(l_extendedprice[3]), sum(l_extendedprice[3]*1-l_discount[7]), sum(l_extendedprice[3]*1-l_discount[7]*1+l_tax[10]), avg(l_quantity[2]), avg(l_extendedprice[3]), avg(l_discount[7]), count(*)(0)
+        Group by: l_returnflag[0], l_linestatus[1]
+        -> PhysicScanTable lineitem (inccost=6001215, cost=6001215, rows=5881190) (actual rows=5914)
```

Comment:
"actual rows = 5914" - this is not right. 

We shall run against an empty TPCH dataset. More precisely, we shall only do EXPLAIN command with tpch1g stats load without actually running it (but we now don't have EXPLAIN implemented @arzuschen, so a workaround is to run emtpy dataset).

Reply:
done